maindesign.css

```diff
@@ -0,0 +1,48 @@
+.header{
+	color:black;
+	font-family: 'Lobster', cursive;
+	font-size:70px;
+	text-align:center;
+	text-decoration:underline;
+}
+
+.subtitle{
+	font-family:'Montserrat', sans-serif;
+	font-size: 25px;
+	text-align:center;
+	color:black
+}
+
+body{
+	background-color:#white
+}
+
+h1{
+	font-family:Courier;
+	font-size:30px;
+	color:white;
+	text-decoration:none;
+	text-align:center;
+}
+
+p{
+	font-family:Courier;
+	font-size:15px;
+	color:white;
+	text-align:center;
+}
+
+.blogpost1 {
+  background-color:#bc1a1a;
+	padding:100px;
+}
+
+.blogpost2 {
+  background-color:#153466;
+	padding:100px;
+}
+
+.blogpost3 {
```

Comment:
It's not exactly a problem to have this as a CSS class, but it's only ever going to apply to one page. So alternatively, what if you have a `blogpost` class that just did the padding (that's the same in each of these classes) and then you added the background color with inline CSS in the file itself. That'd keep this file a bit cleaner, and it'd have the custom part closer to the code that's actually unique to that page.